### PR TITLE
Refactors dna console + creates debug dna disk

### PR DESCRIPTION
**Pull request by jlsnow301**
## About The Pull Request
QoL for the dna console & the coders messing with it.

Console:
- Adds a scrollbar to the mutations list rather than the entire window 
- Spaces the print mutation/injector/save buttons out
- Prevents the mutation names from clipping
- Candystripe tabs, subjectively easier to see

Debug:
- Adds a disk with every human mutation on it. Must be admin spawned in.

## Why It's Good For The Game
Better UX
Ncji59ByYV

## Changelog

:cl:
qol: The genetics console is a bit easier to use with scrolling mutations and tabs.
admin: Added an debug data disk to test mutations.
/:cl:

### CODE REVIEW

**Review by SuperNovaa41 — APPROVED** [review]
code is fine, just get a tgui review

**Comment by jlsnow301** [comment]
a lot of lines come just from my formatter. if this is an issue I can rewrite it to just feature the changes

File: tgui/packages/tgui/interfaces/DnaConsole/MutationInfo.js [hunk 1]
```diff
@@ -141,108 +141,138 @@ export const MutationInfo = (props, context) => {
...
-        {mutation.Source === 'disk' && (
-          <MutationCombiner
-            disabled={!hasDisk
+      <Stack vertical>
+        <Stack.Item>
+          {mutation.Source === 'disk' && (
+            <MutationCombiner
+              disabled={!hasDisk
               || diskCapacity <= 0
               || diskReadOnly}
```
Comment (JohnFulpWillard) [inline tgui/packages/tgui/interfaces/DnaConsole/MutationInfo.js:150]: indentation looks weird here

File: tgui/packages/tgui/interfaces/DnaConsole/MutationInfo.js [hunk 2]
```diff
@@ -141,108 +141,138 @@ export const MutationInfo = (props, context) => {
...
+          )}
+        </Stack.Item>
+        <Stack.Item>
+          <Stack>
+            {['disk', 'occupant'].includes(mutation.Source) && (
+              <Stack.Item>
+                <Button
+                  icon="save"
+                  disabled={savedToConsole
             || !mutation.Active}
```
Comment (JohnFulpWillard) [inline tgui/packages/tgui/interfaces/DnaConsole/MutationInfo.js:223]: here too

File: tgui/packages/tgui/interfaces/DnaConsole/MutationInfo.js [hunk 3]
```diff
@@ -141,108 +141,138 @@ export const MutationInfo = (props, context) => {
...
+            )}
+            {['console', 'occupant'].includes(mutation.Source) && (
+              <Stack.Item>
+                <Button
+                  icon="save"
+                  disabled={savedToDisk
             || !hasDisk
             || diskCapacity <= 0
             || diskReadOnly
             || !mutation.Active}
```
Comment (JohnFulpWillard) [inline tgui/packages/tgui/interfaces/DnaConsole/MutationInfo.js:239]: and here

File: tgui/packages/tgui/interfaces/DnaConsole/MutationInfo.js [hunk 4]
```diff
@@ -141,108 +141,138 @@ export const MutationInfo = (props, context) => {
...
+                  color="red"
+                  content={`Delete from ${mutation.Source}`}
+                  onClick={() => act(`delete_${mutation.Source}_mut`, {
+                    mutref: mutation.ByondRef,
+                  })} />
+              </Stack.Item>
+            )}
+            {(mutation.Class === MUT_EXTRA || !!mutation.Scrambled
         && mutation.Source === 'occupant')
         && (
```
Comment (JohnFulpWillard) [inline tgui/packages/tgui/interfaces/DnaConsole/MutationInfo.js:260]: ditto

**Review by JohnFulpWillard — COMMENTED** [review]
just minor readability thing because i saw it frequently in the file

**Pull request merged**